frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,21 +1,39 @@
 import React, { useEffect, useState } from "react";
-import { getSightings, Sighting } from "../../clients/apiClient";
+import {
+  getExternalApiSightings,
+  getSightings,
+  Sighting,
+  SightingFromExternalApi,
+} from "../../clients/apiClient";
 import { SightingCard } from "./SightingCard";
+import { SightingCardForExternalApi } from "./SightingCardForExternalApi";
 import "./BrowseSightings.scss";
 
 export const BrowseSightings: React.FC = () => {
   const [sightings, setSightings] = useState<Sighting[]>();
+  const [externalApiSightings, SightingFromExternalApi] =
+    useState<SightingFromExternalApi[]>();
 
   useEffect(() => {
     getSightings().then(setSightings);
+    getExternalApiSightings().then(SightingFromExternalApi);
   }, []);
 
-  if (sightings === undefined) {
+  if (sightings === undefined || externalApiSightings === undefined) {
     return <p>Loading</p>;
   }
 
   return (
     <>
+      <h1>Reported Sightings From External Api</h1>
```

Comment:
Semantically, there should only be one `<h1>` per page, and that's the whole title of the page. On this page, the `<h1>` would just be "Sightings", and then both of these would be `<h2>`s
```suggestion
      <h1>Sightings</h1>
      <h2>Reported Sightings From External Api</h2>
```